### ISSUE TITLE
strip ANSI color codes from stack traces

### ISSUE THREAD

**Issue by FND** [complate/complate-fractal#27]
minimal test case:

```jsx
<p>
    <div />
</p>
```

this results in an exception thanks to Prettier's HTML validation:

```
SyntaxError: Unexpected closing tag "p". It may happen when the tag has already been closed by another tag. For more info see https://www.w3.org/TR/html5/syntax.html#closing-elements-that-have-implied-end-tags (1:15)
[0m[31m[1m>[22m[39m[90m 1 | [39m[33m<[39m[33mp[39m[33m>[39m[33m<[39m[33mdiv[39m[33m>[39m[33m<[39m[33m/[39m[33mdiv[39m[33m>[39m[33m<[39m[33m/[39m[33mp[39m[33m>[39m[0m
[0m [90m   | [39m              [31m[1m^[22m[39m[0m
```

I'm note entirely sure where colorization occurs, but we should either avoid that happening in the first place or strip ANSI color codes afterwards (cf. https://github.com/mmalecki/ansispan/blob/master/lib/ansispan.js)

**Comment by FND** [comment]
fixed in feefceebdfcd0170afc05ceb5a4483f05396688c

**Comment by moonglum** [comment]
Great, thanks 🙇
